tests/test_templates.py

```diff
@@ -812,6 +802,10 @@ def circuit(basis_state):
 class TestQSVT:
     """Test the QSVT algorithm."""
 
+    @pytest.mark.skipif(
+        device_name == "lightning.tensor",
+        reason="lightning.tensor does not support BlockEncode",
```

Comment:
We could provide support for BlockEncode by decomposing it to `qml.PrepSelPrep` in a followup PR.

Reply:
yea, will do it when working on all gate support.

Reply:
Awesome!